src/components/App.js

```diff
@@ -0,0 +1,13 @@
+import BooksForm from '../containers/BooksForm';
+import BooksList from '../containers/BooksList';
+
+function App() {
```

Comment:
## Convert to ES6 syntax
- [ ] We need to ensure we are using ES6 syntax all throughout the codebase. Kindly convert this `App` component function to an arrow based function.
```suggestion
const App = () => {
```